powersimdata/design/transmission/zones.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+def calculate_zone_capacity(grid):
+    """For each zone in a grid, calculate the aggreagte zone transmission
+    capacity (in a transport model, ignoring power flow).
```

Comment:
A blank line should be left here

Reply:
<br class="Apple-interchange-newline">I am wondering if the name of the function clearly communicate what is being done. I feel like discarding intra-zone branches should be reflected somehow in the name of the function. What about&nbsp;calculate_interzone_capacity?  

How about just deleting the branch filtering line?

Reply:
It depends what you want to calculate. I guess we have three options:
* sum the `rateA` of all the branches within a zone, i.e, excluding branches that cross zone borders
* sum the `rateA` of all the branches that have one end in the zone
* sum the `rateA` in a zone as you do, i.e. by considering branches that only cross borders

If we want all the above options, we can have thre functions or one function with keyword arguments.